code/game/objects/structures/signs.dm

```diff
@@ -48,7 +48,10 @@
 /obj/item/sign/attackby(obj/item/tool as obj, mob/user as mob)	//construction
 	if(istype(tool, /obj/item/screwdriver) && isturf(user.loc))
 		var/direction = input("In which direction?", "Select direction.") in list("North", "East", "South", "West", "Cancel")
-		if(direction == "Cancel") return
+		if(direction == "Cancel")
+			return
+		if(QDELETED(src))
+			return
 		var/obj/structure/sign/S = new(user.loc)

```

Comment:
Wouldn't it be better to not have it create a new sign? Just set the original sign's direction after screwdrivering

Reply:
This is turning the sign item (that you can hold in your hand) into a sign structure (that is in the world)

Reply:
Aaaaahhhh, right.